screens/ContactScreen.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { View, Text, TextInput, StyleSheet, Button } from 'react-native';
+import styled from 'styled-components/native'
+
+
+const ContactContainer = styled.View`
+  display: flex;
+  align-items: center;
+  height: 100%;
+`;
+
+
+export const Contact = () => {
+  const [Name, onChangeName] = React.useState(null);
```

Comment:
Do you need to write `React.` before `useState`, is it because you don't import `useState` from React? Curious!